### DISCUSSION TITLE
Name of repo

### DISCUSSION THREAD

**Discussion by marijnbent** [wordproof/drupal-module#1] [General]
Any preference for the name of this repo? I think it would be nice to use the same naming convention for every CMS, so after consensus, I will rename the WordPress plugin as well. 

**Proposal**:
`wordproof/cms name + term used by cms for extensions`

**Examples**
_Drupal_: wordproof/drupal-extension
_WordPress_: wordproof/wordpress-plugin

**Comment by marijnbent** [comment]
@bbrala :-)

**Reply by bbrala** [reply]
Already correct now i think.
